labelbox/client.py

```diff
@@ -490,3 +501,11 @@ def create_project(self, **kwargs):
                 any of the attribute names given in kwargs.
         """
         return self._create(Project, kwargs)
+
+    def get_roles(self):
```

Comment:
Can this just be `roles`

and then be a class property?

Reply:
That would be a different access pattern than the other db objects. Although this one behaves differently because it can't be modified. What do you think?